### PR TITLE
Shell script for easy setup of the project

### PR DESCRIPTION
**Pull request by ndkshr**
### Issue Reference
Adds shell script to automate the entire setup for both Apple Mac and Ubuntu Linux

### Summarize
Shell script has options from downloading and installing python to setup of postgresql. It automates and detects the OS and executes the commands.

> Note: Different Linux distros use different package manager, this PR specifically deals with brew and apt only, since they are the most popular ones around.

### CODE REVIEW

File: easy-setup.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+brew install python
+# Install requirements.txt
+pip3 install -r requirements.txt
+pip3 install -r requirements_debug.txt
+# Implementing docker.sh
+chmod 777 docker.sh
+sh docker.sh
+# PostgreSQL : Expecting that brew is preinstalled in your mac
+brew install postgres
+ln -sfv /usr/local/opt/postgresql/*.plist ~/Library/LaunchAgents
```
Comment (frezbo) [inline easy-setup.sh]: Is this not working? `brew services start postgres`?

File: easy-setup.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+BUILD_LIN=$"Linux"
+BUILD_WIN="Windows"
+KERALARESCUE_REPO_LINK="https://github.com/IEEEKeralaSection/rescuekerala.git"
+
+# Commands for Apple Mac
+if [ $OS_BASE == $BUILD_MAC ] 
+then
+# Install Git and the repo for Kerala Rescue
+brew install git
+# ======= Uncomment the following line if you do not have the repository downloaded
```
Comment (frezbo) [inline easy-setup.sh]: The docker installation is missing `brew cask install docker`.

Comment (ndkshr) [reply easy-setup.sh]: Docker is not mandatory, as per my understanding. The docker might repeat the same stuffs again.

File: easy-setup.sh (hunk 1 above)
Comment (ndkshr) [reply easy-setup.sh]: Did not work on my mac the first time, but the commands in this PR will work. Not the straightforward method, but it has a degree of certainty

**Comment by biswaz** [comment]
Duplicate of #1092

**Pull request closed without merging**